lib/commands/connect/export.js

```diff
@@ -15,17 +15,21 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/export';
-    let response = yield api.request(context.auth.password, 'GET', url)
+    let connection, response;
```

Comment:
Though I should also point out that this would be a separate feature, so probably for a different PR.